src/nn/wasm/PreparedModel.js

```diff
@@ -32,13 +33,20 @@ export default class PreparedModel {
    */
   async prepare(model) {
     this._model = model;
+    const modelInputs = model._inputs;
     const operations = model._operations;
     this._nn_ops = await getNNOpsInstance();
 
     this._preference = model._preference;
     this._supportedOps = model._supportedOps;
     this._eager = model._eager;
 
+    if (gemm_context === null && 
+      model._operands[modelInputs[0]].type === OperandCode.TENSOR_QUANT8_ASYMM) {
+        gemm_context = new this._nn_ops.GemmContext;
+        gemm_context.set_max_num_threads(1);
```

Comment:
Now compile `nn_ops.js ` with no threads, so here should set the value to 1.

Reply:
do we really need to expose the GemmContext to JS? It looks like a singleton instance and can stay in native. How about we just expose `set_max_number_threads`?

Reply:
We used the `GemmContext` it in the following uint8 function:
```js
void depthwiseConvUint8Wrapper(const DepthwiseParams& op_params,
                                 const RuntimeShape& inputShape, 
                                 const intptr_t inputData, 	
                                 const RuntimeShape& filterShape, 
                                 const intptr_t filterData, 	
                                 const RuntimeShape& biasShape, 
                                 const intptr_t biasData, 	
                                 const RuntimeShape& outputShape, 
                                 intptr_t outputData,
                                 gemmlowp::GemmContext* gemm_context)

void convUint8Wrapper(const ConvParams& op_params, 
                        const RuntimeShape& inputShape, 
                        const intptr_t inputData, 
                        const RuntimeShape& filterShape, 
                        const intptr_t filterData, 
                        const RuntimeShape& biasShape, 
                        const intptr_t biasData, 
                        const RuntimeShape& outputShape, 
                        intptr_t outputData,
                        const RuntimeShape& im2colShape, 
                        intptr_t im2colData,
                        gemmlowp::GemmContext* gemm_context)

void fullyConnectedUint8Wrapper(const FullyConnectedParams op_params,
                                  const RuntimeShape& inputShape, 
                                  const intptr_t inputData, 
                                  const RuntimeShape& weightsShape, 
                                  const intptr_t weightsData, 
                                  const RuntimeShape& biasShape, 
                                  const intptr_t biasData, 
                                  const RuntimeShape& outputShape, 
                                  intptr_t outputData,
                                  gemmlowp::GemmContext* gemm_context)
```
If we want to expose the tflite ops more originally, maybe we need to expose it to JS? 
If not, I can also declare it as a static global variable in `binding.cpp`, and just expose the function `set_max_number_threads`.
WDYT?